index.bs

```diff
@@ -721,28 +724,24 @@ spec:dom-ls; type:interface; text:Document
   The values of <a link-type="grammar">suborigin-policy-option</a> that may be
   present in a <a>suborigin policy</a> have the following effects:
 
-  * '<dfn>`unsafe-postmessage-receive`</dfn>' When a message is sent <i>to</i> a
```

Comment:
what happened here with the diff? why did it get deleted and then reintroduced below? Were you refactoring the bs file also?